src/openforms/registrations/contrib/objects_api/tests/test_backend.py

```diff
@@ -220,3 +226,199 @@ def test_registration_works_after_failure(self, m: requests_mock.Mocker):
+    VCR_TEST_FILES = Path(__file__).parent / "files"
+
+    def setUp(self):
+        super().setUp()
+
+        config_patcher = patch(
+            "openforms.registrations.contrib.objects_api.models.ObjectsAPIConfig.get_solo",
+            return_value=ObjectsAPIConfig(),
+        )
+        self.mock_get_config = config_patcher.start()
+        self.addCleanup(config_patcher.stop)
+
+        self.objects_api_group = ObjectsAPIGroupConfig.objects.create(
+            objecttypes_service=ServiceFactory.create(
+                api_root="http://localhost:8001/api/v2/",
+                api_type=APITypes.orc,
+                oas="https://example.com/",
+                header_key="Authorization",
+                # See the docker compose fixtures:
+                header_value="Token 171be5abaf41e7856b423ad513df1ef8f867ff48",
+                auth_type=AuthTypes.api_key,
+            ),
+            objects_service=ServiceFactory.create(
+                api_root="http://localhost:8002/api/v2/",
+                api_type=APITypes.orc,
+                oas="https://example.com/",
+                header_key="Authorization",
+                # See the docker compose fixtures:
+                header_value="Token 7657474c3d75f56ae0abd0d1bf7994b09964dca9",
+                auth_type=AuthTypes.api_key,
+            ),
+            drc_service=ServiceFactory.create(
+                api_root="http://localhost:8003/documenten/api/v1/",
+                api_type=APITypes.drc,
+                # See the docker compose fixtures:
+                client_id="test_client_id",
+                secret="test_secret_key",
+                auth_type=AuthTypes.zgw,
+            ),
+            catalogi_service=ServiceFactory.create(
+                api_root="http://localhost:8003/catalogi/api/v1/",
+                api_type=APITypes.ztc,
+                # See the docker compose fixtures:
+                client_id="test_client_id",
+                secret="test_secret_key",
+                auth_type=AuthTypes.zgw,
+            ),
+        )
+
+    def test_submission_with_objects_api_backend_create_and_update_object(self):
+        plugin = ObjectsAPIRegistration(PLUGIN_IDENTIFIER)
+
+        with self.subTest("Create an object"):
+            submission_create = SubmissionFactory.from_components(
+                [
+                    {
+                        "key": "voornaam",
+                        "registration": {
+                            "attribute": RegistrationAttribute.initiator_voornamen,
+                        },
+                    },
+                ],
+                submitted_data={"voornaam": "Foo"},
+                form_definition_kwargs={"slug": "fd-create"},
+            )
+            object_create_result = plugin.register_submission(
+                submission_create,
+                {
+                    "version": 1,
+                    "objecttype": UUID("8e46e0a5-b1b4-449b-b9e9-fa3cea655f48"),
```

Comment:
In https://github.com/open-formulieren/open-forms/pull/4548, I created an objecttype which accepts anything. `8e46e0a5-b1b4-449b-b9e9-fa3cea655f48` still works because I think by default `additionalProperties` is `true`, but it might be better to use the new one I added for clarity (once it gets merged).